homeassistant/components/weatherflow_cloud/icons.json

```diff
@@ -34,6 +34,67 @@
       "lightning_strike_last_epoch": {
         "default": "mdi:lightning-bolt"
       },
+
+      "precip_accum_local_day": {
+        "default": "mdi:umbrella-closed",
+        "range": {
+          "0": "mdi:umbrella-closed",
```

Comment:
```suggestion
```
we can remove this, right? it should fallback to the default if it is bellow `0.01`

Reply:
same for the other entities